tests/unit/states/test_storage_policies.py

```diff
@@ -0,0 +1,409 @@
+from unittest.mock import MagicMock
+from unittest.mock import Mock
+from unittest.mock import patch
+
+import pytest
+import saltext.vmware.modules.storage_policies as storage_policies_module
+import saltext.vmware.states.storage_policies as storage_policies
+from pyVmomi import pbm
+
+
+def mock_with_name(name, *args, **kwargs):
+    # Can't mock name via constructor: https://docs.python.org/3/library/unittest.mock.html#mock-names-and-the-name-attribute
+    mock = Mock(*args, **kwargs)
+    mock.name = name
+    return mock
```

Comment:
**suggestion** This should be in a helpers type file and imported, e.g. `from tests.helpers import mock_with_name` or `import tests.helpers` and then `return tests.helpers.mock_with_name(...)`

Reply:
I don't see package with name tests.helpers
I take this from test_dvportgroup.py

Reply:
I'm OK with not blocking the PR on this - but let's go ahead and create that module and pull mock_with_name out into a common place